gradle.properties

```diff
@@ -26,3 +26,7 @@ org.gradle.jvmargs=-Xmx1280m
 systemProp.https.protocols=TLSv1,TLSv1.1,TLSv1.2
 
 jacocoExclusions=com/linecorp/armeria/internal/common/CurrentJavaVersionSpecific,META-INF/versions/**
+
+# Scala options
+scalaVersions=2.13.3
+defaultScalaVersions=2.13.2
```

Comment:
Question: Why these properties have two different versions?

Reply:
Oops... it was cruft. Actually, I removed [gradle-scala-multiversion-plugin](https://github.com/ADTRAN/gradle-scala-multiversion-plugin) and add `scalapb_2.12` and `scalapb_2.13` because it is difficult to integrate gradle-scala-multiversion-plugin with Armeria's gradle scripts.
And we need a lot of workarounds and some of them are flaky. 😅